fork-off/src/config.rs

```diff
@@ -47,7 +42,54 @@ pub struct Config {
         multiple_occurrences = true,
         takes_value = true,
         value_delimiter = ',',
-        default_value = "Aura,Aleph,Balances,Sudo,Staking,Session,Elections,System.Account"
+        default_value = "Aura,Aleph,Sudo,Staking,Session,Elections"
```

Comment:
I am worried that 
* copying `Balances` from Testnet (`Balances.locks` in particular) might cause local chainspec config to be overwritten from Testnet, and Testnet has different validators than Devnet, which will cause Devnet malfunction. On local run we're creating some `locks` just in the genesis block.
* copying `System.Account` from testnet would cause validators on Devnet to have 0 balance, preventing above mentioned feature to work correctly

Reply:
I think leaving `System.Account` and `Balances` won't hurt - we're copying a lot from Testnet anyway. On the other hand we might get rid of `fork-off` so maybe we should not invest much more time into it. We can pass palelts via GH arguments so we're good.